app/src/main/java/com/sereno/serenoandroidapp/activities/WeatherInfoActivity.kt

```diff
@@ -37,20 +39,30 @@ class WeatherInfoActivity : AppCompatActivity() {
             .addConverterFactory(MoshiConverterFactory.create(moshi))
             .build()
 
+        val cityNameView = findViewById<EditText>(R.id.inputCityName)
+        val cityNameText = cityNameView.text.toString()
+
         thread {
             try {
                 val service: OpenWeatherMapService =
                     retrofit.create(OpenWeatherMapService::class.java)
                 val weatherApiResponse = service.getCurrentWeatherData(
-                    "anan", BuildConfig.OWM_API_KEY, "metric", "ja"
+                    cityNameText, BuildConfig.OWM_API_KEY, "metric", "ja"
                 ).execute().body()
-                    ?: throw IllegalStateException("bodyがnull")
+                    ?: throw IllegalStateException()
```

Comment:
```suggestion
                    ?: throw IllegalStateException("body is null")
```
例外書いてるとはいえ、残しておくべきかと